app/waf/src/models/service.model.ts

```diff
@@ -40,6 +40,12 @@ export class Service extends Entity {
   })
   endpointpolicy: string;
 
+  @property({
+    type: 'string',
+    required: true,
+  })
+  appId: string;
```

Comment:
appId --> applicationId ?

This property name can be exposed by REST API.

Reply:
I will change it in next push